<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMsg</class>
 <widget class="QDialog" name="DialogMsg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Colores primarios*/
/* 
gris oscuro		#222222		#333333		#444444
gris claro 		#999999		#DDDDDD
verde claro						#C8CC8E
vinotinto  						#742427
*/
/*Colores secundarios */
/* 
Azules  		#36C9C6		#00BDB9	#77ACA2
rojos		 	#910D3F		#C70039		#F94646
naranjas 		#D34E24		#F28123		#F7F052
*/

/* Funetes/*

/*
italic:				font: italic 9pt &quot;Ubuntu&quot;;
regular:			font: 9pt &quot;Ubuntu&quot;;
light:				font: 300 9pt &quot;Ubuntu&quot;;
medium:			font: 500 9pt &quot;Ubuntu&quot;;
bold:				font: 700 9pt &quot;Ubuntu&quot;;
*/

/****************************************************************************************************************************************************/



QFrame#frame_dialog{
	background-color: #333333;
	border-radius:15px;
}

QFrame#frame_content{
	background-color: #444444;
	border-radius:5px;
}


QLabel#label_text{
	font: 300 10pt &quot;Ubuntu&quot;;
	color: #DDDDDD;
margin-left: 10px;
}
QTextBrowser#textBrowser_description{
	background-color: transparent;
	font: 300 10pt &quot;Ubuntu&quot;;
	color: #999999;
	margin-left: 20px;
	border: none;
}



QToolButton#toolButton_exit{
background-color: transparent;
}

QPushButton[style_msn_button=&quot;1&quot;]{
font: 500 9pt &quot;Ubuntu&quot;;
}




/*◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘           QPushButton           ◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘◘*/




QPushButton[QPushButtonStyle=&quot;1&quot;]{
font: 500 10pt &quot;Ubuntu&quot;;
color: #222222;
background-color: #77ACA2;
border: none;
padding: 6px 25px;
border-radius: 8px ;
}
QPushButton[QPushButtonStyle=&quot;1&quot;]:hover{
background-color: #36C9C6;
}



/*****************************************/
QPushButton[QPushButtonStyle=&quot;2&quot;]{
font: 500 10pt &quot;Ubuntu&quot;;
color: #DDDDDD;
background-color: #910D3F;
border: none;
padding: 6px 5px;
border-radius: 8px ;
}
QPushButton[QPushButtonStyle=&quot;2&quot;]:hover{
background-color: #C70039;
}



/*****************************************/
QPushButton[QPushButtonStyle=&quot;3&quot;]{
font: 500 10pt &quot;Ubuntu&quot;;
color: #222222;
background-color: #999999;
border: none;
padding: 6px 5px;
border-radius: 8px ;
}
QPushButton[QPushButtonStyle=&quot;3&quot;]:hover{
background-color: #DDDDDD;
}


</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame_dialog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_exit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>9</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../recursos/iconos/iconos_consola/exit.svg</normaloff>../recursos/iconos/iconos_consola/exit.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>15</width>
            <height>15</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_icon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../recursos/iconos/iconos_msg/question.svg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame_content">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_text">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="textBrowser_description">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:300; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Text drescription&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_save">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Guardar</string>
             </property>
             <property name="QPushButtonStyle" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_yes">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Si</string>
             </property>
             <property name="QPushButtonStyle" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_accept">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Aceptar</string>
             </property>
             <property name="style_msn_button" stdset="0">
              <number>1</number>
             </property>
             <property name="QPushButtonStyle" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_notSave">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>No Guardar</string>
             </property>
             <property name="QPushButtonStyle" stdset="0">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_not">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>     No     </string>
             </property>
             <property name="QPushButtonStyle" stdset="0">
              <number>2</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_cancel">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>  Cancelar  </string>
             </property>
             <property name="QPushButtonStyle" stdset="0">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
